Monika After Story/game/script-affection.rpy

```diff
@@ -84,6 +84,26 @@ init -1 python in mas_affection:
     G_NORMAL = -3
 
 init python:
+    #Functions to freeze exp progression for story events, use wisely.
+    def mas_FreezeGoodAffExp():
+        persistent._mas_affection_goodexp_freeze = True
+        
+    def mas_FreezeBadAffExp():
+        persistent._mas_affection_badexp_freeze = True
+        
+    def mas_FreezeBothAffExp():
```

Comment:
Since this function does the same thing as the smaller functions do, just call those functions in here:
```python
def mas_freezeBoth():
    mas_freezeGoodAffExp()
    mas_freezeBadAffExp()
```